lib/availability_status_listener.rb

```diff
@@ -52,7 +52,7 @@ def process_event(event)
     options[:last_available_at] = options[:last_checked_at] if options[:availability_status] == 'available'
 
     object.update!(options)
-    raise_event("#{model_class}.update", object.as_json, event.headers)
+    object.raise_event_for_update(options.keys)
```

Comment:
I'm not sure if the headers will be present (they are assigned to the Insights::Api::Common::Request by Controller, right?)

Reply:
We pass the headers through the kafka event, so yes this won't work with the new `safe_headers` method. 

Not sure how we can get around it other than having the listener just do the manual raise with an if around it or just having a default parameter.

Reply:
have the `safe_headers` as a default value for 2nd param of `raise_event_for_update` seems to be fine to me

Reply:
@slemrmartin Good catch! @lindgrenj6 thanks for advice 👍 

Now fixed 👍